### PR TITLE
Don't set the accept attribute for iOS devices

### PR DESCRIPTION
**Pull request by ClemannD**

### CODE REVIEW

File: projects/rxjs-uploader/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "rxjs-uploader",
-  "version": "1.4.2",
+  "version": "1.4.4",
```
Comment (ClemannD) [inline projects/rxjs-uploader/package.json]: I used 1.4.3 as a beta release, so have to go to 1.4.4

In the future, will do 1.4.3-beta AND tag it as beta, that way it doesn't mess up the release number order

File: projects/rxjs-uploader/src/lib/rxjs-uploader.ts [hunk 2]
```diff
@@ -94,7 +94,12 @@ export class Uploader<FileUploadType extends FileUpload = FileUpload> {
             fileInputElement.setAttribute('multiple', 'multiple');
         }
         if (accept) {
-            fileInputElement.setAttribute('accept', accept);
+            // For iOS, the accept attribute is broken, not allowing some filetypes when they should be allowed.
+            // Solution is to not set the accept attribute if user is on an iOS device.
+            // https://github.com/lionheart/openradar-mirror/issues/19227
+            if (!!navigator.platform && !(/iPad|iPhone|iPod/.test(navigator.platform))) {
```
Comment (dmayerdesign) [inline projects/rxjs-uploader/src/lib/rxjs-uploader.ts]: Nit: would be good to pull this into a little method, but up to you whether it's worth the effort.

File: projects/rxjs-uploader/src/lib/rxjs-uploader.ts [hunk 3]
```diff
@@ -506,9 +516,8 @@ export class Uploader<FileUploadType extends FileUpload = FileUpload> {
             // First, check if the file is under the size limit.
             if (!this._fileSizeLimitMb || file.size < this._fileSizeLimitMb * BYTES_PER_MB) {
               // Then, check if the file type is supported.
-                console.log(`mimeType: ${mimeType}`);
```
Comment (dmayerdesign) [inline projects/rxjs-uploader/src/lib/rxjs-uploader.ts]: 👏

File: projects/rxjs-uploader/src/lib/rxjs-uploader.ts [hunk 4]
```diff
@@ -506,9 +516,8 @@ export class Uploader<FileUploadType extends FileUpload = FileUpload> {
...
                 if (!!this._allowedContentTypes.find((contentType) => contentType === '*')
-                    || this._allowedContentTypes.some((contentType) => mimeType.includes(contentType.replace(/\./g, '')))) {
+                    || this._allowedContentTypes.some((contentType) => contentType === mimeType)) {
```
Comment (dmayerdesign) [inline projects/rxjs-uploader/src/lib/rxjs-uploader.ts]: 👍

**Pull request merged**